_std/macros/chat_output.dm

```diff
@@ -0,0 +1,51 @@
+/* ====== SPAN MACROS ====== */
```

Comment:
Unless there is something preventing that I think these should use string interpolation instead of +.